experimental/dds/tree2/src/test/feature-libraries/sequence-field/sequenceFieldToDelta.spec.ts

```diff
@@ -129,17 +154,29 @@ describe("SequenceField - toDelta", () => {
 			},
 		];
 		assertMarkListEqual(actual, expected);
+		assert.deepEqual(repairDataBuilder.marks, new Map([]));
 	});
 
-	it("delete", () => {
+	it("delete produces repair data", () => {
 		const changeset = Change.delete(0, 10);
-		const mark: Delta.Delete = {
-			type: Delta.MarkType.Delete,
+		const mark: Delta.MoveOut = {
+			type: Delta.MarkType.MoveOut,
+			count: 10,
```

Comment:
I recommend lowering the count to 2 so you don't have to type out a ton of raw data in the test.